yogstation/code/game/objects/items/cards_ids.dm

```diff
@@ -66,6 +66,35 @@
 /obj/item/card/id/captains_spare
 	icon_state = "id_gold"
 
+/obj/item/card/limitedemag
+	name = "flimsy cryptographic sequencer"
```

Comment:
Making this not parented to the emag type means changes in emags won't be reflected over to here automatically. Additionally, doing it this way makes it harder to have emags be capable of emagging limited emags, half-breaking https://github.com/yogstation13/Yogstation-TG/pull/1662.

Make this a subtype of ``/obj/item/card/emag``.

Reply:
the reason i de-parented it is because it conflicted with the emag's afterattack. i try my best to not fuck with tg's code (anymore) so my best solution was to make a new card

Reply:
Could just change the way it does the breaking so you could handle breaking logic after all the other afterattack logic.

Reply:
done